packages/backend/server/src/core/workspaces/resolvers/page.ts

```diff
@@ -543,12 +583,11 @@ export class PagePermissionResolver {
   @Mutation(() => Boolean)
   async revokeDocUserRoles(
     @CurrentUser() user: CurrentUser,
-    @Args('docId') docId: string,
-    @Args('userIds', { type: () => [String] }) userIds: string[]
+    @Args('input') input: RevokeDocUserRolesInput
```

Comment:
This interface is considered an incompatible parameter update, right? How can the old version of the client be compatible?

Reply:
This API is new in 0.20 and has not been released yet